### PR TITLE
MYOB FAQ + Reference

### PR DESCRIPTION
**Pull request by pzaichkina**
# Description

- Joining the FAQ and reference pages for MYOB
- Calling out the integration as synchronous

## Type of change

- [X] New document(s)/updating existing
- [ ] Fixes
- [ ] Styling
- [ ] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: sidebars/integrations.js [hunk 1]
```diff
@@ -60,7 +60,6 @@ module.exports = [
           "integrations/accounting/myob/accounting-myob",
           "integrations/accounting/myob/accounting-myob-setup",
           "integrations/accounting/myob/myob-integration-reference",
-          "integrations/accounting/myob/faq-myob-accountright-live",
```
Comment (mcclowes) [inline sidebars/integrations.js:63]: Probably need a redirect for this deleted page?

File: docs/integrations/accounting/myob/myob-integration-reference.md [hunk 2]
```diff
@@ -30,12 +30,36 @@ Codat's data model does not support negative unit amounts in line items of Direc
 
 Codat does not currently support updating item based invoices.
 
-## Multiple currencies
+## Integration behavior
+
+### Synchronous integration
+
+Our MYOB integration is a synchronous integration. However, for simplicity and consistency with the rest of Codat's integrations, you should treat it as asynchronous.
```
Comment (mcclowes) [inline docs/integrations/accounting/myob/myob-integration-reference.md:37]: Link to read more maybe?

Comment (pzaichkina) [reply docs/integrations/accounting/myob/myob-integration-reference.md:37]: to the CUD page?

Comment (mcclowes) [reply docs/integrations/accounting/myob/myob-integration-reference.md:37]: yes

**Review by mcclowes — APPROVED** [review]

**Pull request merged**